app/src/css/modules/_modals.scss

```diff
@@ -94,9 +94,11 @@
 }
 
 .default-modal {
+  text-align: center;
```

Comment:
This is redundant because we have the `text-align: center;` for `.modal-body` already.